### PR TITLE
Add scripts to populate and drop the messages collection

### PR DESCRIPTION
**Pull request by Be-Like**
Refer #279.

### CODE REVIEW

File: api/db/populate-events.js [hunk 1]
```diff
@@ -12,12 +12,15 @@ const {
   getCountry,
   getRegion,
   getRandomSubset,
+  getArticle,
```
Comment (AlbertoPdRF) [inline api/db/populate-events.js]: I believe that with this change the first line is not needed anymore, right?

File: api/db/populate-messages.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+let displayLogs;
+
+const createMessage = async (messageParams) => {
+  const message = new Message({
+    friendship: messageParams.friendship,
+    user: messageParams.user,
+    content: messageParams.content,
+  });
+  message.save();
```
Comment (AlbertoPdRF) [inline api/db/populate-messages.js]: There's a missing `await` here

File: api/db/populate-messages.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+const createMessages = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating messages collection...');
+  displayLogs = verbose;
+
+  const friendshipsArray = await Friendship.find();
+  const messagesArray = [];
+
+  for (const friendship of friendshipsArray) {
+    const messagesCount = Math.floor(Math.random() * 10);
```
Comment (AlbertoPdRF) [inline api/db/populate-messages.js]: I would say we should make this number bigger and instead make the content of every message smaller, as that could be a better model a real conversation? I'll leave a comment about this where the content of the message is generated too

File: api/db/populate-messages.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+    for (let i = 0; i < messagesCount; i++) {
+      const user =
+        Math.random() < 0.5 ? friendship.requestant : friendship.receivant;
+      const content = getArticle();
```
Comment (AlbertoPdRF) [inline api/db/populate-messages.js]: As said above, I think a real conversation would consist mostly of smaller messages, so maybe we can throw a random number here to decide if we call the `getArticle()` function (should be improbable) or generate just a sentence. How does that sound?

File: api/db/drop.js [hunk 5]
```diff
@@ -125,12 +127,31 @@ const dropFriendships = async (confirmed = false) => {
...
+  }
+};
+
+const dropMessages = async (confirmed = false) => {
+  console.log('\n', '\x1b[0m', 'Dropping messages collection...');
+  if (confirmed || (await getMessagesCount()) !== 0) {
+    await Message.collection.drop();
+    console.log('\x1b[31m', 'Dropped messages collection');
+  } else {
+    console.log('\x1b[31m', 'Messages collection is already empty');
```
Comment (AlbertoPdRF) [inline api/db/drop.js]: The first param should be `'\x1b[33m'`

**Review by AlbertoPdRF — CHANGES_REQUESTED** [review]
This looks mostly good @Be-Like, I've just left a few comments inline. Thanks for your work on another great PR! :tada:

File: api/db/populate-messages.js (hunk 3 above)
Comment (Be-Like) [reply api/db/populate-messages.js]: That's fine. However, I would suggest we reconsider the quantity of users and/or events we are creating. It is beginning to take a very long time to generate all the collections. And most of these items we only really need a handful of records to get a clear idea of how the application will behave.

Comment (AlbertoPdRF) [reply api/db/populate-messages.js]: Yeah, I think we can reduce those by a factor of ten, and if someone needs more they can just use the `-m` argument for that

File: api/db/drop.js [hunk 6]
```diff
@@ -141,9 +141,9 @@ const dropMessages = async (confirmed = false) => {
   console.log('\n', '\x1b[0m', 'Dropping messages collection...');
   if (confirmed || (await getMessagesCount()) !== 0) {
     await Message.collection.drop();
-    console.log('\x1b[31m', 'Dropped messages collection');
+    console.log('\x1b[33m', 'Dropped messages collection');
```
Comment (AlbertoPdRF) [inline api/db/drop.js]: This one was correct as `'\x1b[31m'`, so let's change it back

File: api/db/populate-events.js [hunk 7]
```diff
@@ -71,15 +70,15 @@ const createEvents = async (multiplier, randomLocation, verbose) => {
   const now = new Date();
   const futureDate = new Date().setFullYear(now.getFullYear() + 5);
 
-  for (let i = 0; i < 500 * multiplier; i++) {
+  for (let i = 0; i < 50 * multiplier; i++) {
```
Comment (AlbertoPdRF) [inline api/db/populate-events.js:73]: Please note that this change (and the users' one) requires a small update on the API's README.

File: api/db/populate-messages.js [hunk 8]
```diff
@@ -27,12 +27,12 @@ const createMessages = async (verbose) => {
...
 
   for (const friendship of friendshipsArray) {
-    const messagesCount = Math.floor(Math.random() * 10);
+    const messagesCount = Math.floor(Math.random() * 25);
 
     for (let i = 0; i < messagesCount; i++) {
       const user =
         Math.random() < 0.5 ? friendship.requestant : friendship.receivant;
-      const content = getArticle();
+      const content = getParagraph();
```
Comment (AlbertoPdRF) [inline api/db/populate-messages.js:35]: Maybe we could pass a `Math.ceil(Math.random() * 3)` parameter to this `getParagraph()` function?

**Review by AlbertoPdRF — CHANGES_REQUESTED** [review]
Just 3 more small comments to address and we are good to go! :ship:

File: api/db/populate-messages.js (hunk 8 above)
Comment (AlbertoPdRF) [reply api/db/populate-messages.js:35]: Oops, this comment doesn't make sense, please disregard it

**Review by AlbertoPdRF — APPROVED** [review]
This LGTM, thanks @Be-Like! :100:

**Pull request merged**